habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1298,10 +1312,10 @@ def _apply_velocity_action(
             time_step = self._time_step
 
         # Map velocity actions
-        self.vel_control.linear_velocity = np.array(
+        self.vel_control.linear_velocity = mn.Vector3(
             [0.0, 0.0, -linear_velocity]
         )
-        self.vel_control.angular_velocity = np.array(
+        self.vel_control.angular_velocity = mn.Vector3(
             [0.0, angular_velocity, 0.0]
         )
```

Comment:
Out of curiosity, what difference does this make? Is it just for typing?

Reply:
The angular_velocity and linear_velocity are already in np.array format. When you do np.array([0.0, 0.0, -linear_velocity]), it gives an error. I saw they were using mn.Vector3 in Habitat 2.0 controller so I also tried it and that worked. Basically mn.Vector3 is somehow handling that assertion.